templates/zerver/help/see-if-user-here-away.md

```diff
@@ -0,0 +1,35 @@
+### User List
+
+You can see if a user is active on Zulip by looking at the user list, located on the far right side of your screen.
+
+![User List](/static/images/help/user-list.png)
+
+The **Search people** field at the top of the user list allows you to search for a specific user. As you type the user's name, the user list will shrink, to show only people with that name.
+
+![Search Field](/static/images/help/search-field-user-list.png)
+
+If you mouse over a user's name on the user list, you will see a small chevron(![chevron](/static/images/help/chevron.png)).
+
+Clicking on this chevron brings up a menu like the one below.
+
+![Name Menu](/static/images/help/menu-name-in-user-list.png)
+
+User status is marked by little circles to the left of a user’s name:
+
+ * A green circle
+   (<span class="indicator green solid"></span>) means the
+   user is active or currently on Zulip.
+
+ * A white, circle (<span class="indicator grey"></span>) means the user is
+   not active and was not recently so.
+
+ * A orange half-filled circle
+   (<span class="indicator orange"></span>)
+   means the user was recently active but is not any longer.
```

Comment:
Try "user was recently active" or "user is idle", depending on the message you want to get across.